tests/Unit/Models/WebhookDeliveryTest.php

```
@@ -0,0 +1,151 @@
+        $this->assertFalse(WebhookDelivery::claim($db, 'github', 'abc-123'));
+    }
+
+    #[Test]
+    public function claimReclaimsFailedDeliveryForRetry(): void
+    {
+        $stmt = $this->createMock(PDOStatement::class);
+        $stmt->method('rowCount')->willReturn(1);
+        $queries = [];
+        $db = $this->createMock(Database::class);
+        $db->method('tableExists')->with('webhook_deliveries')->willReturn(true);
+        $db->expects($this->exactly(2))->method('query')->willReturnCallback(function (string $sql, array $params = []) use ($stmt, &$queries): PDOStatement {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$params'. (undefined)

(UnusedFormalParameter)